dolphinscheduler-common/src/main/resources/common.properties

```diff
@@ -58,7 +58,7 @@ fs.s3a.secret.key=OloCLq3n+8+sdPHUhJ21XrSxTC+JK
 yarn.resourcemanager.ha.rm.ids=192.168.xx.xx,192.168.xx.xx
 
 # if resourcemanager HA enable or not use resourcemanager, please keep the default value; If resourcemanager is single, you only need to replace ds1 to actual resourcemanager hostname.
-yarn.application.status.address=http://10.172.33.1:%s/ws/v1/cluster/apps/%s
+yarn.application.status.address=http://ds1:8088/ws/v1/cluster/apps/%s
```

Comment:
Thank you very much for your PR.
The port value in the yarn application url 'yarn.application.status.address' is configured in the 'resource.manager.httpaddress.port' property, So here should be modified to 'yarn.application.status.address=http://ds1:%s/ws/v1/cluster/apps/%s'